### PR TITLE
docs: add Eric JiuRan's blog to showcase

### PR DESCRIPTION
**Pull request by 1084350607**
## Motivation

add userList

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

**Comment by Josh-Cena** [comment]
No need to close this, the status will be updated for you once the CLA check comes through :)

**Comment by Josh-Cena** [comment]
@1084350607 Several notes about your site:

- You forgot to translate the blog. When you are on https://www.siyuanwa.cn/blog/tags/Javascript and you go to the `en` locale, the route doesn't even exist. That may also be a bug on our side, but for now you should at least make an empty `docusaurus-plugin-content-blog` folder.
- English navbar => `Billbill` should be `Bilibili`

Otherwise, nice site 👍

**Comment by 1084350607** [comment]
@Josh-Cena So Appreciated that point out some problems, I'll fix them. There are still lots of work about translate. Thank you so much.

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -503,6 +503,14 @@ const Users: User[] = [
     source: 'https://github.com/eta-dev/eta',
     tags: ['opensource'],
   },
+  {
+    title: 'Eric JiuRan',
+    description: 'Front-end developer blog',
+    preview: require('./showcase/eric.png'),
+    website: 'https://www.siyuanwa.cn/',
+    source: 'https://github.com/1084350607/blog',
+    tags: ['opensource', 'personal'],
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:512]:
```suggestion
    tags: ['opensource', 'personal', 'i18n'],
```

**Pull request merged**